packages/frontend-web/src/app/scenes/Comments/components/CommentDetail/CommentDetail.tsx

```diff
@@ -1120,12 +1142,15 @@ export class CommentDetail extends React.Component<ICommentDetailProps, IComment
 
   @autobind
   handleScoreClick(scoreClicked: ICommentScoreModel) {
+    const thresholdByTag = getTaggingSensitivityForTag(this.props.taggingSensitivitiesInCategory, scoreClicked);
+    const scoresSelectedByTag = this.props.allScores.filter(
+      (score) => score.tagId === scoreClicked.tagId,
+    ).sort((a, b) => a.score > b.score ? -1 : a.score < b.score ? 1 : 0) as List<ICommentScoreModel>;
```

Comment:
This looks like it was just moved; but also seems funny that we're not doing "a.score - b.score" ... ?